shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -144,6 +144,7 @@ class AccessibilityBridge final {
   int32_t previous_route_id_;
   std::unordered_map<int32_t, blink::CustomAccessibilityAction> actions_;
   std::vector<int32_t> previous_routes_;
+  NSString* last_scroll_announcement;
```

Comment:
Nit: ivar names end with an underscore by convention.

Reply:
I don't see the final release for this on the destructor. Just use `scoped_nsobject` instead to prevent the leak.

Reply:
Done, thanks!